Lib/test/test_ipaddress.py

```diff
@@ -2548,6 +2548,24 @@ def testsixtofour(self):
                          sixtofouraddr.sixtofour)
         self.assertFalse(bad_addr.sixtofour)
 
+    # issue41004 Hash collisions in IPv4Interface and IPv6Interface
+
```

Comment:
Please delete this blank line, so that it's more obvious the comment belongs to the following method, not the previous one.